lib/index.js

```diff
@@ -116,30 +119,36 @@ function mergeOptions(options) {
 }
 
 /**
- * Lint each imported component with postcss-bem-linter
- * and stylelint
+ * Lint component with postcss-bem-linter and stylelint
  *
+ * @param {String} css
  * @param {Object} options
+ * @param {String} filename
  * @returns {Promise} Used by postcss-import transform
  */
-function lintImportedFiles(options, css, filename) {
+function lintComponent(css, options, filename) {
```

Comment:
I would call this `lintFile` because it lints utils, entry file and other stuff too right?